src/peft/tuners/lora/layer.py

```diff
@@ -145,6 +145,9 @@ def __init__(self, base_layer: nn.Module, ephemeral_gpu_offload: bool = False, *
         elif hasattr(base_layer, "W_q") and base_layer.__class__.__name__ == "HQQLinear":
             # HQQ layers
             in_features, out_features = base_layer.in_features, base_layer.out_features
+        elif base_layer.__class__.__name__.startswith("Patched"):
+            # INC layers
+            in_features, out_features = base_layer.in_features, base_layer.out_features
```

Comment:
Is this not already covered by the `else` clause below?

Reply:
It looks like its covered, but we get this warning:
```bash
/root/peft/src/peft/tuners/lora/layer.py:157: UserWarning: Unsupported layer type '<class 'neural_compressor.torch.algorithms.fp8_quant._quant_common.helper_modules.PatchedLinear'>' encountered, proceed at your own risk.
```
With added `elif` we avoid the `Unsupported later type` warning.